src/main/resources/fabric.mod.json

```diff
@@ -4,17 +4,18 @@
   "version": "${version}",
   "name": "Bow Infinity Fix",
   "description": "Removes need to have an arrow in your inventory to use the Infinity I enchant on your bow.",
-  "authors": ["Parker8283"],
+  "authors": ["Parker8283","MeeniMc"],
   "contact": {
     "homepage": "https://www.curseforge.com/minecraft/mc-mods/bow-infinity-fix",
     "sources": "https://github.com/Parker8283/BowInfinityFix",
     "issues": "https://github.com/Parker8283/BowInfinityFix/issues"
   },
   "license": "LGPLv3",
+  "icon": "assets/bif/bif-logo-light.png",
   "environment": "*",
   "mixins": ["bif.mixins.json"],
   "depends": {
-    "fabricloader": ">=0.4.0",
-    "minecraft": "1.16.x"
+    "fabricloader": ">=0.11.6",
```

Comment:
From what I remember (and correct me if I'm wrong), the Fabric Loader is independent of the MC version. So realistically, someone could use newer loader versions on older MC versions, and vice versa. Now, whether or not that is advisable is another question :P

So, my question, is there anything from newer loader versions that I utilize that would require me to _need_ to uptick this?

Reply:
There is a minimum version of fabric loader that supports 1.17. I know for a fact that 0.11.3 works, and maybe some prior 0.11.x also work. 0.4.0 absolutely doesn't. The  change in Gradle is important because it is needed to have `gradle runClient` working; this one is not strictly needed since the predicate will necessarily be satisfied for any 1.17 deployment.

Reply:
Huh, the more you know. Alrighty, then this is fine. I just try not to uptick minimum required versions unless absolutely necessary. Mostly from my experience in the wider software industry, and people's affinity to stick with old versions of software :P